CMakeLists.txt

```diff
@@ -72,6 +72,7 @@ add_definitions(-DSUBSURFACE_SOURCE="${SUBSURFACE_SOURCE}")
 #
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 ")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu99 -Winconsistent-missing-override")
```

Comment:
shouldn't this be `-Wno-inconsisten-missing-overrides`? Or am I completely misunderstanding how this works?